dace/frontend/python/memlet_parser.py

```diff
@@ -3,17 +3,27 @@
 import copy
 import re
 from collections import namedtuple
-from typing import Any, Dict, Union
+from typing import Any, Dict, List, Optional, Tuple, Union
+from dataclasses import dataclass
 
-from dace import data, subsets
+from dace import data, dtypes, subsets
 from dace.frontend.python import astutils
 from dace.frontend.python.astutils import rname
 from dace.memlet import Memlet
-from dace.symbolic import pystr_to_symbolic
+from dace.symbolic import pystr_to_symbolic, SymbolicType
 from dace.frontend.python.common import DaceSyntaxError
 
 MemletType = Union[ast.Call, ast.Attribute, ast.Subscript, ast.Name]
-MemletExpr = namedtuple('MemletExpr', ['name', 'accesses', 'wcr', 'subset'])
+
+
+@dataclass
```

Comment:
We should add this to the developer documentation (adds default __init__ and __repr__?)

Reply:
what do you mean?

Reply:
The @dataclass decorator.

Reply:
Oh, that's a Python built-in now.